pennylane/vqe/vqe.py

```diff
@@ -45,6 +46,22 @@ class Hamiltonian:
 
     .. seealso:: :class:`~.ExpvalCost`, :func:`~.molecular_hamiltonian`
 
+    .. Warning::
```

Comment:
@Lucaman99 very clear warning message 🙂 

However, I would suggest moving it _below_ the example, since the reader might have a better understanding of the context after viewing the example.

Reply:
For sure!